src/csharp/Microsoft.Spark/Hadoop/FS/FileSystem.cs

```diff
@@ -0,0 +1,58 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using Microsoft.Spark.Hadoop.Conf;
+using Microsoft.Spark.Interop;
+using Microsoft.Spark.Interop.Ipc;
+
+namespace Microsoft.Spark.Hadoop.FS
```

Comment:
```suggestion
namespace Microsoft.Spark.Hadoop.Fs
```
? Since we use `Ipc` instead of `IPC`?